src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerReleaseAndTransferService.java

```diff
@@ -752,6 +788,7 @@ public InmateDetail courtTransferIn(final String offenderNo, final RequestForCou
 
         final MovementReason movementReason = getMovementReason(requestForCourtTransferIn.getMovementReasonCode(), latestExternalMovement.getMovementReason());
 
+        offenderBooking.setStatusReason(MovementType.CRT.getCode() + "-" + movementReason.getCode());
```

Comment:
minor bug in existing transfer from court, status reason was never being set so remained at the reason set during the transfer out to court. 
It would only be noticeable if they change the court reason when returning back from court (which I guess WPIP never does)